itou/users/models.py

```diff
@@ -515,10 +515,8 @@ def has_valid_approval(self):
         return self.latest_approval and self.latest_approval.is_valid()
 
     @property
-    def has_common_approval_in_waiting_period(self):
-        return (self.latest_approval and not self.latest_approval.is_valid()) or (
-            self.latest_pe_approval and not self.latest_pe_approval.is_valid()
-        )
+    def is_latest_common_approval_in_waiting_period(self):
```

Comment:
nit: j'aurais laissé du `has_*`

```suggestion
    def has_latest_common_approval_in_waiting_period(self):
```

Reply:
Oui, user has ...